src/main/java/com/ibm/etcd/client/EtcdClient.java

```diff
@@ -386,8 +387,10 @@ private CallCredentials refreshCredentials() {
             private Metadata tokenHeader; //TODO volatile TBD
             private final long authTime = System.currentTimeMillis();
             private final ListenableFuture<Metadata> futureTokenHeader =
-                    Futures.transform(authenticate(),
-                            (Function<AuthenticateResponse,Metadata>)ar -> tokenHeader = tokenHeader(ar));
+                    Futures.transform(
+                            authenticate(),
+                            (Function<AuthenticateResponse,Metadata>)ar -> tokenHeader = tokenHeader(ar),
```

Comment:
this cast can also be removed now (and import of `Function` class removed)

Reply:
Done